ci-operator/step-registry/ipi/conf/vsphere/zones/multisubnets/ipi-conf-vsphere-zones-multisubnets-commands.sh

```diff
@@ -20,6 +20,12 @@ declare vsphere_extra_portgroup_2
 source "${SHARED_DIR}/vsphere_context.sh"
 # shellcheck source=/dev/null
 source "${SHARED_DIR}/govc.sh"
+declare -a vips
+mapfile -t vips < "${SHARED_DIR}"/vips.txt
+APIVIP=${vips[0]}
```

Comment:
yeah, I was thinking it at the beginning. However if so we also need to set `.platform.vsphere.loadBalancer.type` which value should be `UserManaged` in this case for 4.16,  not sure `.platform.vsphere.loadBalancer.type` will be supported in previous version.  so I did not change the origin install-config.yaml.  to avoid to cause the conflict. thus I don't need to regression test on other profile, WDYT?

Reply:
That should be not difficult.
```
declare -a vips
mapfile -t vips < "${SHARED_DIR}"/vips.txt
if [[ ${LB_TYPE} == "UserManaged" ]]; then
    APIVIP=${vips[0]}
    INGRESSVIP=${vips[1]}
    LB_TYPE_DEF="loadBalancer:
      type: UserManaged"
else
    APIVIP=""
    INGRESSVIP=""
    LB_TYPE_DEF=
fi

cat >>"${CONFIG}" <<EOF
...
platform:
  vsphere:
    apiVIP: ${APIVIP}
    ingressVIP: ${INGRESSVIP}
    ${LB_TYPE_DEF}
```

Reply:
apiVIPs also need to instead of `apiVIP`. 

how about
```
if [ ${LB_TYPE} = "UserManaged" ]; then
  yq --inplace eval-all 'select(fileIndex == 0) * select(fileIndex == 1)' "${CONFIG}" - <<< "
platform:
  baremetal:
    apiVIPs:
    - ${vips[0]}
    ingressVIPs:
    - ${vips[1]}
    loadBalancer:
      type: UserManaged
"      
fi  
```

Reply:
Do you means here it is updating `platform.vsphere.apiVIPs`, but not `platform.vsphere.apiVIP`? `apiVIPs` is newly introduced in this feature?  If update  `platform.vsphere.apiVIPs`, we still need to leave `platform.vsphere.apiVIP` to be empty? Do you know what is the difference between them?

Actually the initial purpose of the comment is not recommend to use `yq`, because it is download yq binary from internet, when multi jobs are running, it will hit network bottleneck issue.

Reply:
no, aipVIPs is not new, guess they already used for long time maybe from dualstack. apiVIP should be deprecated in future.

Reply:
https://github.com/openshift/installer/blob/master/pkg/asset/installconfig/vsphere/validation.go I did not see any validation for apiVIP..

Reply:
Sounds like we should drop the line `platform.vsphere.apiVIP` in install-config.yaml, but use the new one.

If using `platform.vsphere.apiVIPs`, we can do the same:
```
declare -a vips
mapfile -t vips < "${SHARED_DIR}"/vips.txt
if [[ ${LB_TYPE} == "UserManaged" ]]; then
    APIVIPS_DEF="apiVIPs:
      - ${vips[0]}"
    INGRESSVIPS_DEF="ingressVIPs:
      - ${vips[1]}"
    LB_TYPE_DEF="loadBalancer:
      type: UserManaged"
else
    APIVIPS_DEF=""
    INGRESSVIPS_DEF=""
    LB_TYPE_DEF=""
fi

cat >>"${CONFIG}" <<EOF
...
platform:
  vsphere:
    ${LB_TYPE_DEF}
    ${APIVIPS_DEF}
    ${INGRESSVIPS_DEF}
```

Reply:
ok, thanks.  let me try